gui-libs/display-manager-init/files/display-manager.confd-4

```diff
@@ -0,0 +1,11 @@
+# We always try and start X on a static VT. The various DMs normally default
+# to using VT7. If you wish to use the xdm init script, then you should ensure
+# that the VT checked is the same VT your DM wants to use. We do this check to
+# ensure that you haven't accidentally configured something to run on the VT
+# in your /etc/inittab file so that you don't get a dead keyboard.
+CHECKVT=7
```

Comment:
I do not think this will really work with -suid, or if X is not started as root. The login's TTY is used with *logind. Have you tested it with elogind?

Reply:
Yes, I've been using this with elogind and without even using X actually.

My whole setup is running on wayland with elogind.